portal/src/main/webapp/css/header.css

```diff
@@ -149,6 +149,18 @@ header {
     width:100%;
 }
 
+.legacy .modal-header .close span {
+    display:none;
+}
+
+.legacy .modal-header .close:before {
+    content: "✖";
```

Comment:
You actually cannot put HTML (including HTML entities) into css inserted content.  And it won't break because cbioportal code never gets built in jitpack, so not vulnerable to jitpack's weird corruption issue.